xbmc/cores/VideoPlayer/Edl.h

```diff
@@ -22,6 +22,7 @@
 
 #include <string>
 #include <vector>
+#include "FileItem.h"
```

Comment:
No need to include FileItem.h, a forward declaration of class CFileItem is sufficient here. Please keep dependencies as minimalistic as actually technically needed.